ihatemoney/models.py

```diff
@@ -94,6 +94,18 @@ def members_stats(self):
             for member in self.active_members
         ]
 
+    @property
+    def monthly_stats(self):
+        """Compute expenses by month
+
+        :return: a dict of years mapping to a dict of months mapping to the amount
+        :rtype dict:
+        """
+        monthly = defaultdict(lambda: defaultdict(int))
```

Comment:
amount seems to be [a float](https://github.com/spiral-project/ihatemoney/blob/1a657ae67a77b11e1126a07cb470b563b29788b1/ihatemoney/models.py#L367) rather than an int.

Reply:
It doesn't really matter, since it is only used for setting the initial value, which is 0.
And, to be exact, it should be a `Decimal`, but that will be for the future :)

Reply:
You are totally correct, it's better not to confuse types. Changed it to a float :)